app/javascript/new_design/select2.js

```diff
@@ -122,12 +122,14 @@ const adresseOptions = {
       };
     },
     processResults(data) {
+      let r = data.features.map(({ properties: { label }, geometry }) => ({
+        id: label,
+        text: label,
+        geometry
+      }));
+      r.unshift({ id: data.query, text: data.query });
```

Comment:
Ça te va si on ajoute un commentaire pour expliquer ce qu'on fait ?

```suggestion
      // Allow the user to select an arbitrary address missing from the results,
      // by adding the plain-text query to the list of results.
      r.unshift({ id: data.query, text: data.query });
```